sites/fast-website/src/app/components/navigation/navigation.template.ts

```diff
@@ -0,0 +1,10 @@
+import { html, slotted } from "@microsoft/fast-element";
+import { Navigation } from "./navigation";
+
+export const NavigationTemplate = html<Navigation>`
+    <template>
+        <nav>
+            <slot ${slotted("items")}></slot>
```

Comment:
add `start` and `end` slots here.

Reply:
This will allow you place the logo and other items inside the navigation component.

![image](https://user-images.githubusercontent.com/37851214/81843321-4e0a6a00-9502-11ea-95f0-2f867ee152cc.png)

Reply:
added